js/src/analysis/getTitlePlaceholder.js

```diff
@@ -0,0 +1,26 @@
+/* global wpseoPostScraperL10n, wpseoTermScraperL10n */
+
+var isUndefined = require( 'lodash/isUndefined' );
+
+/**
+ * Returns the title placeholder for use in the title forms.
+ *
+ * @returns {string}
+ */
+function getTitlePlaceholder() {
```

Comment:
Wouldn't we want to tell the function which type of title we are looking for: Post or Term?
Avoiding any unwanted `title_template` to be returned ever.